akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -91,20 +99,28 @@ private[http2] class Http2ServerDemux(http2Settings: Http2CommonSettings, initia
       override def settings: Http2CommonSettings = http2Settings
       override def isUpgraded: Boolean = upgraded
 
+      def maxConcurrentStreams: Int = http2Settings.maxConcurrentStreams
```

Comment:
Maybe we should call that

```suggestion
      def localMaxConcurrentStreamsLimit: Int = http2Settings.maxConcurrentStreams
```